sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -1113,6 +1167,16 @@ export interface FullTextPolicy {
     fullTextPaths: FullTextPath[];
 }
 
+// Warning: (ae-internal-missing-underscore) The name "FullTextStatistics" should be prefixed with an underscore because the declaration is marked as @internal
```

Comment:
should we prefix it? Also, why is it exported?

Reply:
Fixed to remove internals